src/couch/priv/stats_descriptions.cfg

```diff
@@ -290,6 +302,30 @@
     {type, histogram},
     {desc, <<"duration of validate_doc_update function calls">>}
 ]}.
+{[couchdb, query_server, acquired_processes], [
+    {type, counter},
+    {desc, <<"number of acquired external processes">>}
+]}.
+{[couchdb, query_server, process_starts], [
+    {type, counter},
+    {desc, <<"number of os process starts">>}
```

Comment:
Does "os" refer to the operating system here?  If yes, it should be spelled as "OS" instead -- I see that the format allows using abbreviations like that (e.g. LRU)

Reply:
it does, and agree OS would be clearer, unless existing stats say "os".

Reply:
Good point. It was OS at first, but then I noticed the surrounding descriptions were all lower-cased so went with that pattern, higher up in the file we do have a few  "HTTP" and "CouchDB" so "OS" would work. I'll change it.